register/views.py

```diff
@@ -15,12 +16,46 @@ def register(request):
             user = form.save(commit=True)
             email = form.cleaned_data.get("email")
             password = form.cleaned_data.get("password1")
+            first_name = form.cleaned_data.get("first_name")
+            last_name = form.cleaned_data.get("last_name")
+            user.is_candidate = True
+            user.is_employer = False
             user.save()
+            profile = CandidateProfile(
+                first_name=first_name, last_name=last_name, email=email
+            )
+            profile.save()
+            candidate = Candidate(user=user, candidate_profile=profile)
+            candidate.save()
             user = authenticate(email=email, password=password)
             login(request, user)
             messages.success(request, "Account created successfully")
             return HttpResponseRedirect(reverse("candidate_login:candidate_login"))
-
     else:
         form = CandidateRegistrationForm()
-    return render(request, "register/register.html", {"form": form})
+    return render(request, "register/candidate_register.html", {"form": form})
+
+
+def employer_register(request):
+    if request.user.is_authenticated:
+        return HttpResponseRedirect(reverse("uplyft:index"))
+    if request.method == "POST":
+        form = EmployerRegistrationForm(request.POST)
+        if form.is_valid():
+            user = form.save(commit=True)
+            email = form.cleaned_data.get("email")
+            password = form.cleaned_data.get("password1")
+            department = form.cleaned_data.get("department")
+            user.is_employer = True
+            user.is_candidate = False
```

Comment:
I'm just thinking if we need 2 boolean fields to check the user type. 
Wouldn't it probably be easier if we just had one boolean field for it? Or is there a reason why we're storing 2 boolean fields is_employer and is_candidate?